Src/LiquidProjections/Liblog.cs

```diff
@@ -1694,7 +1694,7 @@ internal static bool IsLoggerAvailable()
                     entryType.GetPropertyPortable("Categories"),
                     Expression.ListInit(
                         Expression.New(typeof(List<string>)),
-                        typeof(List<string>).GetMethodPortable("Add", typeof(string)),
+                        typeof(List<string>).GetMethodPortable("Record", typeof(string)),

```

Comment:
The new ProjectReference format doesn't yet support content-only NuGet packages. It's now a bug.